arrow-avro/src/codec.rs

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-use crate::schema::{Attributes, ComplexType, PrimitiveType, Record, Schema, TypeName};
+use crate::schema::{Attributes, ComplexType, Enum, PrimitiveType, Record, Schema, TypeName};
```

Comment:
I don't think you need to being in `Enum` for this.

```suggestion
use crate::schema::{Attributes, ComplexType, PrimitiveType, Record, Schema, TypeName};
```

Reply:
Good point. Removing this import and cleaning up relevant unneeded code.